lib/src/components/NxSubmitMask/NxSubmitMask.scss

```diff
@@ -7,30 +7,21 @@
 @import '../../scss-shared/_nx-variables';
 @import '../../scss-shared/_nx-container-helpers';
 @import '../../scss-shared/_nx-text-helpers';
+@import '../../scss-shared/_nx-mask-helpers';
 
 .nx-submit-mask {
-  align-items: center;
+  @include background-mask(0);
 
-  background-color: transparentize(#555555, 0.25);
-
-  display: flex;
-  justify-content: center;
   position: absolute;
-
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-
-  z-index: $nx-submit-mask-z-index;
 }
 
 .nx-submit-mask--fullscreen {
+  @include background-mask($nx-submit-mask-z-index);
+
   position: fixed;
+  justify-content: left;
```

Comment:
`start` would be less confusing than `left` here

Reply:
> `start` would be less confusing than `left` here

"less confusing"? https://github.com/sonatype/sonatype-react-shared-components/pull/330/commits/c15fb39292d5247bd885798ff4bb8bd6700429eb